test/functional/feature_block.py

```diff
@@ -214,17 +199,12 @@ def get_tests(self):
         b12 = self.next_block(12, spend=out[3])
         self.save_spendable_output()
         b13 = self.next_block(13, spend=out[4])
-        # Deliver the block header for b12, and the block b13.
-        # b13 should be accepted but the tip won't advance until b12 is delivered.
-        yield TestInstance([[CBlockHeader(b12), None], [b13, False]])
-
         self.save_spendable_output()
-        # b14 is invalid, but the node won't know that until it tries to connect
-        # Tip still can't advance because b12 is missing
-        self.next_block(14, spend=out[5], additional_coinbase_value=1)
-        yield self.rejected()
+        b14 = self.next_block(14, spend=out[5], additional_coinbase_value=1)
+        self.sync_blocks([b12, b13, b14], False, 16, b'bad-cb-amount', reconnect=True)
```

Comment:
The comment above `(b12 added last)` is no longer true with this change.

Reply:
yes - removed